lib/Runtime/Base/ThreadContext.cpp

```diff
@@ -2728,7 +2728,7 @@ ThreadContext::InvalidateInlineCacheList(InlineCacheList* inlineCacheList)
         }
     }
     NEXT_SLISTBASE_ENTRY;
-    inlineCacheList->Clear();
+    Adelete(&this->inlineCacheThreadInfoAllocator, inlineCacheList);
```

Comment:
why do you need to invalidate ever inlineCache in inlinecachelist if you are going to delete the inlinecachelist?